src/bridge/jsHelpers.ts

```diff
@@ -42,7 +42,21 @@ import type { Result, GetAddressOptions } from "../hw/getAddress/types";
 import { open, close } from "../hw";
 import { withDevice } from "../hw/deviceAccess";
 
-export type GetAddressesFromPubkey = (pubkey: string) => Promise<string[]>;
+// Customize the way to iterate on the keychain derivation
+type IterateResult = ({
+  transport: Transport,
+  index: number,
+  derivationsCache: Object,
+  derivationScheme: string,
+  derivationMode: DerivationMode,
+  currency: CryptoCurrency,
+}) => Promise<Result>;
+
+type IterateResultBuilder = ({
```

Comment:
```suggestion
export type IterateResultBuilder = ({
```
This type needs to be visible for other coins to customize it.